node-via-nvm/README.md

```diff
@@ -0,0 +1,58 @@
+---
+display_name: node-via-nvm
+description: Install node via nvm
+icon: ../.icons/node.svg
+maintainer_github: coder
```

Comment:
```suggestion
maintainer_github: TheZoker
```
Should we keep this module as official? We should keep as few modules as official as we can easily maintain.